first-bot/command-with-arguments.md

```diff
@@ -116,17 +116,17 @@ Another way to use arguments, when the command should target a specific user \(o
 
 In the context of the `message` event handler, all mentions in a message are part of the `message.mentions` object. This object then contains multiple [Collections](../understanding/collections.md) of different mention types. Here are the various available mention types:
 
-* `message.mentions.members` contains all @mention as [GuildMember](https://discord.js.org/#/docs/main/stable/class/GuildMember) objects.
-* `message.mentions.users` contains all @mention as [User](https://discord.js.org/#/docs/main/stable/class/User) objects.
-* `message.mentions.roles` contains all @role mention as [Role](https://discord.js.org/#/docs/main/stable/class/Role) objects.
-* `message.mentions.channels` contains all \#channel mentions as [TextChannel](https://discord.js.org/#/docs/main/stable/class/TextChannel) or [VoiceChannel](https://discord.js.org/#/docs/main/stable/class/TextChannel) objects.
+* `message.mentions.members` contains all @mention as [GuildMember](https://discord.js.org/#/docs/discord.js/stable/class/GuildMember) objects.
+* `message.mentions.users` contains all @mention as [User](https://discord.js.org/#/docs/discord.js/stable/class/User) objects.
+* `message.mentions.roles` contains all @role mention as [Role](https://discord.js.org/#/docs/discord.js/stable/class//Role) objects.
```

Comment:
docs have all been moved to the `discordjs/docs` repo, this updates the links and also the Link to VoiceChannel was wrong, was directing to TextChannel